packages/astro/test/units/dev/collections-renderentry.test.js

```diff
@@ -101,7 +101,7 @@ describe('Content Collections - render()', () => {
 				assert.equal($('ul li').length, 3);
 
 				// Rendered the styles
-				assert.equal($('style').length, 1);
+				assert.equal($('style').length, 2);
```

Comment:
I can see that we updated the injected styles in different tests. However, I don't see a test where we actually check its content. Did I miss it, perhaps?

Reply:
Done!